src/notification/pushover.ts

```diff
@@ -0,0 +1,23 @@
+/* eslint @typescript-eslint/no-var-requires: "off" */
+import {Config} from '../config';
+import {Logger} from '../logger';
+const Push = require('pushover-notifications');
```

Comment:
Does pushover have es6 imports? Maybe @types?

Reply:
I don't believe so. I couldn't find anything on the [npmjs](https://www.npmjs.com/package/pushover-notifications) page or any examples on github using anything other than `require`. Also tried some workarounds in the code and nothing would work.

I also don't have a ton of experience with all the ES6 standards, so I could be missing something.

Reply:
No worries. Ya, we can create a type definition (look at `src/types`) and then use the

```typescript
import push = require('pushover-notification')
```

Reply:
Thanks, should be fixed now.